lnrpc/routerrpc/router.proto

```diff
@@ -203,6 +199,12 @@ message Failure {
 
     /// A failure type-dependent flags value.
     uint32 flags = 7;
+
+    /**
+    The position in the path of the intermediate or final node that generated
+    the failure message. Position zero is the sender node.
+    **/
+    int32 failure_source_index = 8;
```

Comment:
should we be consistent in naming, `error source ` or `failure source`?

Reply:
Yes good point. I renamed everything to failure. That is what it is called in the spec and also prevents confusion with internal errors.